server/website/website/models.py

```diff
@@ -162,6 +162,7 @@ class Session(BaseModel):
 
     target_objective = models.CharField(
         max_length=64, default=target_objectives.default())
+    hyper_parameters = models.TextField(default="{}")
```

Comment:
1. Rename 'hyper_parameters' --> 'hyperparameters'
2. The default should be a JSON string with the settings from constants.py.
3. Can you prefix the hyperparameter names with the algorithm that uses them? Or, in the case where multiple algorithms use a given setting, make sure the name is meaningful?
For example:
BATCH_SIZE --> GPR_BATCH_SIZE
NUM_THREADS --> TF_NUM_THREADS